### PR TITLE
Micro version updates

### PR DESCRIPTION
**Pull request by rsvoboda**
Micro version updates

 smallrye-jwt can't be updated atm, https://github.com/quarkusio/quarkus/issues/1514 filed to track that problem (change in constructor)

### CODE REVIEW

**Review by Sanne — APPROVED** [review]

**Comment by Sanne** [comment]
Approved as it looks good in principle. @rsvoboda could you judge those CI failures and decide if you want to merge as-is or possibly re-trigger builds to have it check again? the Maven timeouts should be improved now.

**Comment by gsmet** [comment]
@rsvoboda could you rebase and force push? Hopefully the CI failures will go away with the Google Maven Central mirror.

**Comment by rsvoboda** [comment]
@gsmet I triggered new runs from azure pipeline view

**Comment by gsmet** [comment]
@rsvoboda that might not be sufficient if you don't have the Google Maven Central mirror commit. That's why I asked for a rebase.

**Comment by rsvoboda** [comment]
There are test failures in agroal extension, might be H2 update related, kinda weird.
I'm at home now without my work machine, will update PR on Monday.

**Comment by sberyozkin** [comment]
Sorry guys for a mess up with smallrye-jwt, I'm on it. (FYI, I'm not aware of any 1.1.1 changes which bring anything else apart from my change and the dependabout updates). Quarkus jwt principal will become more compact after the update

**Comment by rsvoboda** [comment]
OK, now I have working environment on my home machine too.

Had to drop H2 upgrade to make the tests pass.
And Vert.x upgrade had conflict with io.smallrye.reactive:smallrye-axle-generator:0.0.1.
Will create issues for both.

File: bom/pom.xml [hunk 1]
```diff
@@ -1562,7 +1563,7 @@
             <dependency>
                 <groupId>io.smallrye.reactive</groupId>
                 <artifactId>smallrye-converter-api</artifactId>
-                <version>${smallrye-reactive-streams-operators.version}</version>
+                <version>${smallrye-converter-api.version}</version>
```
Comment (gsmet) [inline bom/pom.xml]: Good catch.

**Pull request merged**